Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -3341,6 +3363,75 @@ def remote_script_automate_results(client: Client, args: dict):
     return_results(run_polling_command(client=client, cmd="sentinelone-remote-script-automate-results", args=args))
 
 
+def get_columns_from_result(columns: list):
+    heading = []
+    for column in columns:
+        name = column.get("name")
+        heading.append(name)
+    return heading
```

Comment:
```suggestion
    return [column.get("name") for column in columns if column.get("name"))
```

If each column must have a name, its better to use column["name"] and not .get()